yjit_codegen.c

```diff
@@ -26,6 +26,9 @@ codeblock_t* cb = NULL;
 static codeblock_t outline_block;
 codeblock_t* ocb = NULL;
 
+// Code for exiting back to the interpreter
+static void *interp_exit;
+
```

Comment:
// Code for exiting back to the interpreter in the leave instruction
static void *leave_exit;